website/blog/2022-04-27-async-indexing.md

```diff
@@ -0,0 +1,213 @@
+---
+title: "Asynchronous Indexing using Hudi"
+excerpt: "How to setup Hudi for asynchronous indexing"
+author: codope 
+category: blog
+---
+
+In its rich set of asynchronous table services, Hudi has added yet another asynchronous service for indexing. It allows users to create different kinds of indexes (files, bloom
+filters, and column stats) on the Hudi table without blocking ingestion.
+
+<!--truncate-->
+
+## Introduction
+
+The metadata table in Hudi is an internal Merge-on-Read (MOR) table that has a single partition called `files` which stores the data partitions to files index that is used for
+efficient file listing. In the release version 0.11.0, we added support for multi-modal indexes which include two other types of indexes like `COLUMN_STATS` and `BLOOM_FILTERS`.
+Multi-modal indexes will greatly improve the record lookup time. Head over to that [blog](https://todo.add.link/) to learn more about multi-modal indexes. In this blog, we discuss how these indexes
```

Comment:
@codope  link for multi modal indexes to be added later ?

Reply:
yeah, made this a wip pr. multi-modal index will be a separate blog and we plan to land that first.